src/Contracts/Time/Time.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/**
+ * This file is part of the Zest Framework.
+ *
+ * @author   Muhammad Umer Farooq <lablnet01@gmail.com>
+ * @author-profile https://www.facebook.com/Muhammadumerfarooq01/
+ *
+ * For the full copyright and license information, please view the LICENSE
+ *  file that was distributed with this source code.
+ *
+ * @since 3.0.0
+ *
+ * @license MIT
+ */
+
+namespace Zest\Contracts\Time;
+
+interface Time
+{
+
+	public function now();	
```

Comment:
What kinds of time manipulation methods/works should be implemented?

Can you organize and define them?

Reply:
Updated you may see now.